### PR TITLE
Update cli_install.php

### PR DESCRIPTION
**Pull request by TheCartpenter**
Added phpdoc titles on methods.

### CODE REVIEW

File: upload/install/cli_install.php [hunk 1]
```diff
@@ -123,6 +123,13 @@ public function index(): void {
 		$this->response->setOutput($output);
 	}
 
+	/**
+	 * Install
+	 * 
+	 * @param $argv
```
Comment (AJenbo) [inline upload/install/cli_install.php:129]: This is invalid without a type

Comment (TheCartpenter) [reply upload/install/cli_install.php:129]: Ok, closing commit. Will re-open a new one.

Comment (AJenbo) [reply upload/install/cli_install.php:129]: You can also update a branch by adding more commits to it to adjust and update the pr, but if you feel it's easier to make a new one that is also OK

Comment (TheCartpenter) [reply upload/install/cli_install.php:129]: I created a new PR: https://github.com/opencart/opencart/pull/13293 . All passed PHP versions.

**Pull request closed without merging**